src/tablist.js

```diff
@@ -123,6 +125,16 @@ SideTabList.prototype = {
       return;
     }
     if (e.which == 2 && SideTab.isTabEvent(e, false)) {
+      e.preventDefault();
+      return;
+    }
+  },
+  onAuxClick(e) {
+    if (!e.target) {
```

Comment:
This is not needed since isTabEvent already checks target (not your fault, I forgot to remove the uses of this elsewhere).